crates/rpc/rpc/src/trace.rs

```diff
@@ -310,6 +338,7 @@ where
                         .into_localized_transaction_traces(tx_info);
                     Ok(Some(traces))
                 },
+                Some(highest_idx),
```

Comment:
this argument should be moved and should be the second argument, so that the closure is always the last argument